composer.json

```diff
@@ -13,7 +13,7 @@
         "php": "^7.1",
         "contao/core-bundle": "4.5.*",
         "contao/installation-bundle": "4.5.*",
-        "contao/manager-plugin": "^2.2",
+        "contao/manager-plugin": "dev-feature/plugin-cache as 2.3.9999",
```

Comment:
Before we merge this, we should merge contao/manager-plugin#13 and change this to `^2.3`.

Reply:
Yes, and release 2.3 version of `contao/manager-plugin`